spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,54 @@
+describe 'user stories' do
+  # As a customer
+  # So that I can check if I want to order something
+  # I would like to see a list of dishes with prices
+
+  it 'so customers can place an order, they can view a menu with dishes and prices' do
+    tas = TakeawayService.new
+    menu = Menu.new
+    expect { tas.view_menu }.to output(menu.view).to_stdout
+  end
+
+  # As a customer
+  # So that I can order the meal I want
+  # I would like to be able to select some number of several available dishes
+
```

Comment:
Your tests would be more readable if you grouped them according to a contexts. It would also allow you to DRY it out by putting the 3 repeated lines into a before method that would execute before the beginning of each test:`before(:each) do end`. Also, instead of creating a new instance of `TakeawayService.new`, you can use RSpec's `subject` which does the same thing.